docs_source_files/content/guidelines_and_recommendations/page_object_models.zh-cn.md

```diff
@@ -1,39 +1,19 @@
 ---
-title: "Page object models"
+title: "PO设计模式"
 weight: 1
 ---
 
-{{% notice info %}}
-<i class="fas fa-language"></i> 页面需要从英语翻译为简体中文。
-您熟悉英语与简体中文吗？帮助我们翻译它，通过 pull requests 给我们！
-{{% /notice %}}
-
-Page Object is a Design Pattern which has become popular in test automation for
-enhancing test maintenance and reducing code duplication. A page object is an
-object-oriented class that serves as an interface to a page of your AUT. The
-tests then use the methods of this page object class whenever they need to
-interact with the UI of that page. The benefit is that if the UI changes for
-the page, the tests themselves don’t need to change, only the code within the
-page object needs to change. Subsequently all changes to support that new UI
-are located in one place.
-
-The Page Object Design Pattern provides the following advantages:
-
-* There is a clean separation between test code and page specific code such as
-  locators (or their use if you’re using a UI Map) and layout.
-* There is a single repository for the services or operations offered by the page
-  rather than having these services scattered throughout the tests.
-
-In both cases this allows any modifications required due to UI changes to all
-be made in one place. Useful information on this technique can be found on
-numerous blogs as this ‘test design pattern’ is becoming widely used. We
-encourage the reader who wishes to know more to search the internet for blogs
-on this subject. Many have written on this design pattern and can provide
-useful tips beyond the scope of this user guide. To get you started, though,
-we’ll illustrate page objects with a simple example.
-
-First, consider an example, typical of test automation, that does not use a
-page object:
+
+PO（page objec）设计模式是一种易于维护和减少代码的设计模式 ，在自动化中已经变得流行。在自动化测试中，PO对象是作为一个与页面交互的接口。 测试中需要与页面的UI进行交互时，便调用该PO对象的方法。这样做的好处是，所有UI中的改变支持都位于同一个位置。如果页面的UI发生了更改，但是测试用例本身不需要更改，只需更改PO中的代码即可。
```

Comment:
HI @ymang714549348,

Thanks for the PR. Can you please make sure to have a line with a max of 70-80 characters. You can spilt into multiple line if possible.

Thanks,
Harsha

Reply:
I spilted it inyo multiple lines

Reply:
sorry,my problem, i will pull the whole repository,and run it ,but it will take some times

Reply:
Hi Ymangz,
How about the punctuation？Is that better if we keep the format in whole page?
Seems there are full width character with external space in your translated text.
e.g.

> 口。 测

> 式 ，在

> 法。这

Reply:
i will modify this to like other page of translated